kalite/project/settings/screenshots.py

```diff
@@ -0,0 +1,27 @@
+"""
+This module is for running a screenshot server (internal FLE purpose mainly).
+To use it, run kalite with --settings=kalite.project.settings.screenshots or
+set then environment variable DJANGO_SETTINGS_MODULE to
+'kalite.project.settings.screenshots'.
+"""
```

Comment:
This will probably affect building the docs, since the screenshot directive attempts to run a management command without specifying a settings module. Is `--settings` honored by all management commands? I'll try this out.